src/main/java/org/mule/module/apikit/Router.java

```diff
@@ -194,13 +214,37 @@ public void setName(String name) {
     this.name = name;
   }
 
-  private CoreEvent buildSubflowEvent(CoreEvent parent, ValidRequest request, String outboundHeadersMapName) {
+  private CoreEvent buildSubFlowEvent(boolean disableValidations, CoreEvent parent,
+                                      ValidRequest request, String outboundHeadersMapName) {
+
     CoreEvent.Builder eventBuilder = CoreEvent.builder(parent);
     eventBuilder.addVariable(outboundHeadersMapName, new HashMap<>());
 
     Message.Builder messageBuilder = Message.builder(parent.getMessage());
-    messageBuilder.value(request.getBody().getPayload());
+    messageBuilder.value(makeRepeatable(disableValidations, request.getBody().getPayload(), parent));
     messageBuilder.attributesValue(request.getAttributes());
     return eventBuilder.message(messageBuilder.build()).build();
   }
+
+  private TypedValue<Object> makeInputBodyRepeatable(boolean disableValidations, CoreEvent event,
+                                                     TypedValue<Object> inputBody) {
+    return new TypedValue<>(makeRepeatable(disableValidations, inputBody.getValue(), event),
+                            inputBody.getDataType(),
+                            inputBody.getByteLength());
+  }
+
+
+  private Object makeRepeatable(boolean disableValidations, Object body, CoreEvent event) {
```

Comment:
No need to pass diasbleValidation as parameter, configuration is reachable from here